src/policy/packages.h

```diff
@@ -89,8 +89,9 @@ bool IsChildWithParents(const Package& package);
  */
 bool IsChildWithParentsTree(const Package& package);
 
-/** Get the hash of these transactions' wtxids, concatenated in lexicographical order (treating the
- * wtxids as little endian encoded uint256, smallest to largest). */
+/** Get the hash of the concatenated wtxids of transactions, with wtxids
+ * treated as a little-endian numbers and sorted in ascending numeric order.
```

Comment:
```suggestion
 * treated as little-endian numbers and sorted in ascending numeric order.
```

Reply:
re: https://github.com/bitcoin/bitcoin/pull/31596#discussion_r1901981925

Thanks, fixed in a followup branch.